src/physics/cam/clubb_mf.F90

```diff
@@ -1239,6 +1249,11 @@ subroutine integrate_mf( nz,
            awwup(k) = awwup(k) + upa(k,i)*upw(k,i)*upw(k,i)
            awwdn(k) = awwdn(k) + dna(k,i)*dnw(k,i)*dnw(k,i)
 
+           auwup(k) = auwup(k) + upa(k,i)*upu(k,i)*upw(k,i)
+           auwdn(k) = auwdn(k) + dna(k,i)*dnu(k,i)*dnw(k,i)
+           avwup(k) = avwup(k) + upa(k,i)*upv(k,i)*upw(k,i)
+           avwdn(k) = avwdn(k) + dna(k,i)*dnv(k,i)*dnw(k,i)
+

```

Comment:
Can we add the sum of the up and dn fluxes to be the variables ```awu``` and ```awv```? Delete them at line 1265/1266., and insert them near line 1279 nearby ```aww``` and ```aw```, as:

```
awu(k) = auwup(k)+ auwdn(k)
awv(k) = avwup(k)+ avwdn(k)
```